zip-####.rst

```diff
@@ -297,20 +364,25 @@ and the Major Grant Review Committee SHOULD utilize these in its funding
 decisions.
 
 
-Direct-grant option
```

Comment:
Just checking that it was intentional to remove the direct-grant option?

Reply:
Yes, the direct grant option section is obsolete to this ZIP.  It seems obsolete more broadly also, considering the strong community support of NDFM.